cmd/system-probe/modules/security_runtime.go

```diff
@@ -0,0 +1,15 @@
+package modules
+
+import (
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	secmodule "github.com/DataDog/datadog-agent/pkg/security/module"
+)
+
+// SecurityRuntime - Security runtime Factory
+var SecurityRuntime = api.Factory{
+	Name: "security_runtime",
+	Fn: func(cfg *config.AgentConfig) (api.Module, error) {
+		return secmodule.NewModule(cfg)
+	},
+}
```

Comment:
Should this module be attached by the security agent and not the system-probe ?

Reply:
The runtime security agent is made of 2 parts :
- one system-probe module for the eBPF code
- an other attached by the security agent that listens for events from the first part and send events to Datadog